crates/types/protobuf/restate/cluster.proto

```diff
@@ -65,6 +65,7 @@ message PartitionProcessorStatus {
   uint64 num_skipped_records = 8;
   ReplayStatus replay_status = 9;
   optional restate.common.Lsn last_persisted_log_lsn = 10;
+  optional restate.common.Lsn last_archived_log_lsn = 12;
```

Comment:
Would it be useful to also include a last-archived-timestamp here?

Reply:
Not sure if it adds more value. I would leave it out.